asciidoc/courses/graphql-basics/modules/2-queries/lessons/3-graphql-relationships/lesson.adoc

```diff
@@ -272,7 +272,7 @@ type ActedIn @relationshipProperties {
 [source,GraphQL]
```

Comment:
the link below:
`https://neo4j.com/docs/graphql-manual/current/type-definitions/relationships/#_relationship_properties`
should be:
`https://neo4j.com/docs/graphql/current/types/relationships/#_relationship_properties`